### ISSUE TITLE
Type error in line 18

### ISSUE THREAD

**Issue by nliviu** [mongoose-os-apps/example-shadow-js#2]
The line `state.on = obj.state.on;` should be `state.on = obj.on;`, otherwise a `type error` is triggered.
The object received by the callback is eg. `{"on":true}`

**Comment by sysdist** [comment]
correct, it should be `state.on = obj.on;` to properly handle the shadow update key
